src/app/portlets/dot-service-integration/dot-service-integration-configuration/dot-service-integration-configuration-list/dot-service-integration-configuration-list.component.ts

```diff
@@ -0,0 +1,112 @@
+import {
+    Component,
+    OnInit,
+    ViewChild,
+    ElementRef,
+    Input,
+    Output,
+    EventEmitter
+} from '@angular/core';
+import { DotServiceIntegrationSites } from '@shared/models/dot-service-integration/dot-service-integration.model';
+import { DotAlertConfirmService } from '@services/dot-alert-confirm';
+
+import { LazyLoadEvent } from 'primeng/primeng';
+import { DotMessageService } from '@services/dot-messages-service';
+import { take } from 'rxjs/operators';
+
+@Component({
+    selector: 'dot-service-integration-configuration-list',
+    templateUrl: './dot-service-integration-configuration-list.component.html',
+    styleUrls: ['./dot-service-integration-configuration-list.component.scss']
+})
+export class DotServiceIntegrationConfigurationListComponent implements OnInit {
+    @ViewChild('searchInput')
+    searchInput: ElementRef;
+
+    @Input() disabledLoadDataButton: boolean;
+    @Input() itemsPerPage: number;
+    @Input() siteConfigurations: DotServiceIntegrationSites[];
+
+    @Output() loadData = new EventEmitter<LazyLoadEvent>();
+    @Output() addConfiguration = new EventEmitter<boolean>();
```

Comment:
add, edit, delete, we already are in a configuration component, we dont need the sufix here.